.github/workflows/ci.yml

```diff
@@ -40,7 +40,9 @@ jobs:
     - name: flake8
       run: pipenv run flake8
     - name: pylint
-      run: pipenv run pylint src
+      run: |
+        pipenv run pylint src test
+        pipenv run pylint -d duplicate-code,invalid-name test/templates/*.py
```

Comment:
I still needed to disable these warnings for scripts in test/templates because otherwise my version of pylint (Ubuntu 20.04) was complaining about invalid case of variables (he wants them all uppercase) and some code duplication.

Reply:
Which version of pylint you are using? Maybe we should force it to specific version in Pipfile instead because of this. I mean if you get different warnings than me.

Reply:
And also are these errors coming up in CI pipeline?

Reply:
I have
```
yugr@yugr-VirtualBox:~/src/my/asm2cfg$ pipenv run pylint --version
pylint 2.12.2
astroid 2.9.3
Python 3.8.10 (default, Nov 26 2021, 20:14:08) 
[GCC 9.3.0]
```
I'm not sure how to go about these differences, maybe require a minimum Pylint version if pipenv supports this.

Reply:
Ah sorry the previous output doesn't even contain the version output. Here is mine:
```
~/programming/asm2cfg (tests/5*) » pylint --version                                                                                                                                                                                                                                                                                                      127 ↵ kazhuu@kazhuu
pylint 2.12.2
astroid 2.9.3
Python 3.10.1 (main, Dec 18 2021, 23:53:45) [GCC 11.1.0]
```
It seems we are using the same version. Wonder where you get the errors. What is the specific line you use to run pylint?